packages/fxa-auth-server/lib/pushbox/index.ts

```diff
@@ -134,7 +140,9 @@ module.exports = function (log, config, statsd, DB = PushboxDB) {
   });
 
   // pushbox expects this in seconds, not millis.
-  const maxTTL = Math.round(config.pushbox.maxTTL / 1000);
+  const maxTTL = Math.round(
+    (config.pushbox.maxTTL as unknown as number) / 1000
+  );
 
   return {
```

Comment:
Instead of returning an object that is like a class instance, we could just have a class, instantiate it, and return an instance of it.

Reply:
What are the advantages of that?

Reply:
Generally the type comes out a little more consistent as its a class instance instead of an ad-hoc dynamic object. One could even easily register it with typedi as well.

Reply:
Can we wait until we get rid of the Pushbox service api code first?  Then we don't need to spend time dealing with the `store` and `retrieve` from `createBackendServiceAPI`.